### DISCUSSION TITLE
Is possible to obtain a particle waves effect?

### DISCUSSION THREAD

**Discussion by Deca** [matteobruni/tsparticles#1790] [Q&A] [answered]
Hi all!

I've played with the interactive demo and looked at the documentation, but I can't manage to have a sort of particles wave/sea

Basically having the particles floating on a plane and creating a sense of depth, like the image below...is that possible?
![image](https://user-images.githubusercontent.com/3716783/125176428-86bfd500-e1d3-11eb-95c2-7194b5dc8bb0.png)

**Comment by matteobruni** [comment, marked as answer]
There's no blur effect, but you can add particles manually using the configuration or using the provided methods.

These are generated particles: https://codepen.io/matteobruni/pen/GRpYdGv

These are manual particles: https://codepen.io/matteobruni/pen/YzpOZpq

And some depth effect using a fake z index can be seen here: https://codepen.io/matteobruni/pen/rNmMVEx

Anyway a real 3D effect is not possible since this library uses the 2D canvas. Animating in the 3rd dimension will require a lot of resources

**Reply by Deca** [reply]
Sorry for late reply, and thank for the suggestions...I'll look into it. For now I'll try to work with the depth effect and blurring. For what I found three.js is one of the libraries that support 3D but is probably too resources intensive for my needs.
